### PR TITLE
Paas 234 handle some errors

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-234

Short description:

### CODE REVIEW

File: backup.yml [hunk 1]
```diff
@@ -67,52 +67,72 @@ actions:
         cluster_role=$(awk -F'=' '$1=="JEL_ENV_ROLE" {print $2}' /metadata_from_HOST); export cluster_role
         env_region=$(awk -F'=' '$1=="JEL_REGION" {print $2}' /metadata_from_HOST | sed 's/[[:punct:]]//g'); export env_region
         cd paas_jelastic_backup
-        tar -czvf digital-factory-data.tar.gz /data/digital-factory-data/
+        tar -czvf digital-factory-data.tar.gz -C / /data/digital-factory-data/
```
Comment (laurentfufu) [inline backup.yml:70]: didn't get the point here

File: backup.yml [hunk 2]
```diff
@@ -67,52 +67,72 @@ actions:
...
+            type: error
+            message: "An error occured during jahia data backup."
     - cmd [proc]: |-
         export AWS_ACCESS_KEY_ID="${settings.aws_access_key}" AWS_SECRET_ACCESS_KEY="${settings.aws_secret_key}"
         provider=$(awk -F'=' '$1=="JEL_CLOUDPROVIDER" {print $2}' /metadata_from_HOST); if [ "$provider" != "aws" ]; then aws_region='eu-west-1'; else aws_region=$(awk -F'=' '$1=="JEL_REGION" {print $2}' /metadata_from_HOST); fi; export AWS_DEFAULT_REGION="$aws_region"
         cluster_role=$(awk -F'=' '$1=="JEL_ENV_ROLE" {print $2}' /metadata_from_HOST); export cluster_role
         env_region=$(awk -F'=' '$1=="JEL_REGION" {print $2}' /metadata_from_HOST | sed 's/[[:punct:]]//g'); export env_region
         cd paas_jelastic_backup
-        tar -czvf digital-factory-config.tar.gz /opt/tomcat/conf/digital-factory-config/
+        tar -czvf digital-factory-config.tar.gz -C / /opt/tomcat/conf/digital-factory-config/
```
Comment (laurentfufu) [inline backup.yml:83]: same

File: backup.yml (hunk 1 above)
Comment (Ronan-Gorain) [reply backup.yml:70]: If not present, it throws a message on stderr and make error detection impossible for this command block

File: backup.yml (hunk 2 above)
Comment (Ronan-Gorain) [reply backup.yml:83]: same :)

**Pull request merged**